srcpkgs/vscode/REMOVE

```diff
@@ -7,6 +7,12 @@ if [ "$UPDATE" = "no" ]; then
             if [ -d usr/share/code-oss ]; then
                 rm -rf usr/share/code-oss
             fi
+            if [ -e usr/share/applications/code-oss.desktop ]; then
+                rm -f usr/share/applications/code-oss.desktop
+            fi
+            if [ -e usr/share/pixmaps/code-oss.png ]; then
+                rm -f usr/share/pixmaps/code-oss.png
+            fi
```

Comment:
why do you remove them manually?

Reply:
Couldn't it be that they still exist from an installation before switching from /usr/share to /usr/lib?

Reply:
yes but xbps-install should remove them on update

Reply:
Ahh okay. I didn't know that. Will fix it asap.

Reply:
Done.